### ISSUE TITLE
generate wrong pictures

### ISSUE THREAD

**Issue by sunkymepro** [microsoft/DisentangledFaceGAN#25]
Dear author，thanks for your great work，it‘s very helpful for me
But when I run the generate_images.py,I get very strange result
![019_01](https://user-images.githubusercontent.com/81243784/118399249-da5be980-b68e-11eb-8974-3f9551508a96.png)
I download the pretrain .pkl file，and set --model=’network-snapshot-020126.pkl’ and =network-snapshot-020126.pkl
The former result is added,the latter result is a blank picture.
I think the reason of this phenomenon is not loading the pkl files fail or loading wrong file.
Is there something needs to notice I miss?I really want to know the answer.

**Comment by YuDeng** [comment]
Hi, can you try not use --model argument and run the generate_images.py using the default setting? The code should download the pretrained model automatically.

**Comment by grinawiem** [comment]
Dear author, I'm studying your code about remaking the face with a new method. It's an amazing work ( DisentangledFaceGAN) in the field of information technology. Thank you for this wonderful and innovative effort. And here I am trying to study and understand your new strategy so that I can mention your work in my research that I am preparing, but when I tried to test it, it became clear to me that I had failed in creating the data correctly. Could you please help me and explain to me more about how to prepare it. Because when I try to teach the model, this result appears on the computer screen.
Please help me;
![Screenshot from 2021-08-13 13-13-35](https://user-images.githubusercontent.com/11414496/129478494-f69404a9-50c7-402d-924a-ccfed779121f.png)
![Screenshot from 2021-08-13 13-13-20](https://user-images.githubusercontent.com/11414496/129478498-3fbf400c-9cb8-4fc3-8949-79b23ffba0b1.png)

**Comment by YuDeng** [comment]
> Dear author, I'm studying your code about remaking the face with a new method. It's an amazing work ( DisentangledFaceGAN) in the field of information technology. Thank you for this wonderful and innovative effort. And here I am trying to study and understand your new strategy so that I can mention your work in my research that I am preparing, but when I tried to test it, it became clear to me that I had failed in creating the data correctly. Could you please help me and explain to me more about how to prepare it. Because when I try to teach the model, this result appears on the computer screen.
> Please help me;
> ![Screenshot from 2021-08-13 13-13-35](https://user-images.githubusercontent.com/11414496/129478494-f69404a9-50c7-402d-924a-ccfed779121f.png)
> ![Screenshot from 2021-08-13 13-13-20](https://user-images.githubusercontent.com/11414496/129478498-3fbf400c-9cb8-4fc3-8949-79b23ffba0b1.png)

Hi, it is strange to get NAN loss when training the VAE. What is your training data and what is the scale of your dataset? If there is too few data, it might fail to train a proper VAE.

**Comment by Yitian-Li** [comment]
> Dear author, I'm studying your code about remaking the face with a new method. It's an amazing work ( DisentangledFaceGAN) in the field of information technology. Thank you for this wonderful and innovative effort. And here I am trying to study and understand your new strategy so that I can mention your work in my research that I am preparing, but when I tried to test it, it became clear to me that I had failed in creating the data correctly. Could you please help me and explain to me more about how to prepare it. Because when I try to teach the model, this result appears on the computer screen. Please help me; ![Screenshot from 2021-08-13 13-13-35](https://user-images.githubusercontent.com/11414496/129478494-f69404a9-50c7-402d-924a-ccfed779121f.png) ![Screenshot from 2021-08-13 13-13-20](https://user-images.githubusercontent.com/11414496/129478498-3fbf400c-9cb8-4fc3-8949-79b23ffba0b1.png)

you should check your data path because you are using a dataset of size "0"

**Comment by chinmayjog13** [comment]
@YuDeng I am facing the same issue when I provide the model argument. If I don't, I face the issue referenced in #33 . I am using python 3.6 and the correct versions of packages given in the readme. Please suggest a solution

**Comment by Yitian-Li** [comment]
已收到您的邮件

**Comment by fak111** [comment]
![image](https://github.com/microsoft/DiscoFaceGAN/assets/97438502/2a26832d-5c84-476b-acd6-6b64f413d96d)
i get wrong picture too

**Comment by Yitian-Li** [comment]
已收到您的邮件

**Comment by fak111** [comment]
i hava got the answer  ,because of the environment. if u choose the another machine , do the again, maybe solove

**Comment by fak111** [comment]
> i hava got the answer ,because of the environment. if u choose the another machine , do the again, maybe solove

or because the computer has enough  size of gpu  ,,,model can't do well in weak computer

**Comment by aurelianocyp** [comment]
我在3090跑的结果和你一样，不太对。换成了 2080 Ti就成功了

**Comment by jadechip** [comment]
Has anyone a solution to this? I working in an environment based off of the 
tensorflow/tensorflow:1.13.2-gpu-py3-jupyter docker image, similar to the recommended specs and this is how my images look:  ![000_00](https://github.com/microsoft/DiscoFaceGAN/assets/5690384/6d0f7d3d-290c-41cc-98bd-efa201ce36f9)

I have tried using multiple different GPUs.

Thank you.

**Comment by Yitian-Li** [comment]
已收到您的邮件
